text/chapter3.md

```diff
@@ -560,20 +604,60 @@ apply f x = f x
 infixr 0 apply as $
 ```
 
-So `apply` takes a function and a value and applies the function to the value. The `infixr` keyword is used to define `($)` as an alias for `apply`.
+So `apply` takes a function and a value and applies the function to the value. The `infixr` keyword is used to define `$` as an alias for `apply`.
 
 But why would we want to use `$` instead of regular function application? The reason is that `$` is a right-associative, low precedence operator. This means that `$` allows us to remove sets of parentheses for deeply-nested applications.
 
 For example, the following nested function application, which finds the street in the address of an employee's boss:
 
 ```haskell
-street (address (boss employee))
+_.street (_.address (_.boss employee))
 ```
 
 becomes (arguably) easier to read when expressed using `$`:
 
 ```haskell
-street $ address $ boss employee
+_.street $ _.address $ _.boss employee
+```
+
+Note that neither of the above examples is idiomatic PureScript. Real-world code is more likely to express this as:
+```haskell
+(boss employee).address.street
+```
+or
+```haskell
+_.boss.address.street employee
+```
+
+There are situations where putting a prefix function in an infix position as an operator leads to more readable code. One example is the `mod` function:
+```text
+> mod 8 3
+2
+```
+This is fine, but doesn't line up with common usage. Wrapping a prefix function in backticks (\`) lets you use a prefix function in infix position as an operator, e.g.,
+```text
+> 8 `mod` 3
+2
+```
+Likewise, wrapping an operator in parentheses lets you use it as a function in prefix position:
+```text
+> 8 + 3
+11
+
+> (+) 8 3
+11
+```
+This allows for compact definitions of curried (or partially applied) functions based on infix operator functions, such as the `add2` function below:
+```text
+> add2 = (+) 2
```

Comment:
Readers might be asking: "Isn't this both equivalently compact, and easier to type?"
```purs
add2 = (+) 2
```

Reply:
I see what you mean. Maybe focusing on the inline function use case would help?

Reply:
Oops, I forgot edit that snippet that I copied.

Not sure if your follow-up comment still applies.

Reply:
Less so, now that you've clarified that you meant that `add 2` would work as well.

What do you think? Are there some better examples of use cases? Or just delete this bit and just go on to operator sections?

(By the way, thanks for the detailed review!)

Reply:
The use cases are pretty obscure, since you can always substitute the original function name. I guess if you wanted to use `composeKleisliFlipped` in a map, backwards fish `(<=<)` is nicer. Not a great example for Ch3 though.

If an explanation isn't necessary for understanding the exercise code, then it's probably safe to skip. Unfortunately, if a beginner encounters something like `(+)` elsewhere, they might have a tough time finding the [corresponding reference](https://github.com/purescript/documentation/blob/master/language/Syntax.md#operators-as-values), or even making sense of the terse explanation. To address the former, I'd like to make a [syntax lookup table](https://github.com/purescript/documentation/issues/344#issuecomment-724250814).

We also don't have a well-defined policy on the book's strategy and scope. There are opportunities to [reduce duplication](https://github.com/purescript-contrib/purescript-book/issues/149#issuecomment-721281643), but I also think it's useful to elaborate on concepts from the spec with concrete examples that involve the chapter code (for example with your explanation of property accessors).

And I appreciate all your help with improving the book as well.